samples/ingestion/ingestion-client/FetchTranscription/TranscriptionProcessor.cs

```diff
@@ -65,6 +49,22 @@ public TranscriptionProcessor(IServiceProvider serviceProvider, IStorageConnecto
             {
                 this.databaseContext = this.serviceProvider.GetRequiredService<IngestionClientDbContext>();
             }
+
+            var serviceBusClientFactory = serviceProvider.GetRequiredService<IAzureClientFactory<ServiceBusClient>>();
```

Comment:
Should we not inject `IAzureClientFactory<ServiceBusClient>` instead of calling `GetRequiredService` ? It can cause issues when we need to mock ServiceBusClient for tests
Let me know your thoughts

Reply:
makes sense - fixed!